### DISCUSSION TITLE
Bind Specific Side of Margin

### DISCUSSION THREAD

**Discussion by DasLixou** [AvaloniaUI/Avalonia#13479] [Q&A]
I want to bind only the right side of margin to a templatedbinding. How can i achieve this?

**Comment by DasLixou** [comment]
So far i tried
```xaml
<Image.Margin>
  <Thickness Right="{TemplatedBinding Righty}" />
</Image.Margin>
```
and
```xaml
<Image Margin="0, {TemplatedBinding Righty}, 0, 0">
</Image>
```

**Comment by rabbitism** [comment]
You can use a converter to convert the source thickness to the target side thickness you want.

**Reply by DasLixou** [reply]
But later i want two values, so i need a thing that converts binding a and binding b to `0, a, 0, b`

**Reply by workgroupengineering** [reply]
MultiBinding with IMultiValueConverter

**Reply by DasLixou** [reply]
ughhh ill try

**Reply by timunie** [reply]
https://github.com/AvaloniaUI/Avalonia.Samples/tree/main/src/Avalonia.Samples/MVVM/ValueConversionSample

**Reply by robloo** [reply]
Yes, there are also existing converters in Avalonia for this we used for the Fluent controls: `MarginMultiplierConverter`, `CornerRadiusFilterConverter`. You can use them for reference.
